svf-llvm/lib/SVFIRBuilder.cpp

```diff
@@ -230,6 +230,8 @@ void SVFIRBuilder::initialiseNodes()
                 iter->second, argval->getArgNo(), icfgNode,
                 llvmModuleSet()->getCallGraphNode(argval->getParent()),iter->first->getType(),
                 LLVMUtil::isArgOfUncalledFunction(argval));
+            if (!argval->hasName())
+                pag->getGNode(iter->second)->setName(std::to_string(argval->getArgNo()));
```

Comment:
What is our previous way for setname of an argument?>

Reply:
Name and sourceloc are set in the same way as other vars: https://github.com/SVF-tools/SVF/blob/42570b418a78f15bc6219256cb243edd7df99da0/svf-llvm/lib/LLVMModule.cpp#L1331

Reply:
what is the difference between `val->getName().str()` and `argval->getArgNo()`? We shall use the former one?

Reply:
we use argval->getArgNo() only when the arg does not have name and val->getName().str() otherwise.

Reply:
It is consistent with prior design: https://github.com/SVF-tools/SVF/blob/42570b418a78f15bc6219256cb243edd7df99da0/svf-llvm/lib/LLVMModule.cpp#L294-L295

Reply:
what is the string value of the `getName` for a typical argval?  Is that "arg no."? We need to align the format.